lib/spreewald_support/mail_finder.rb

```diff
@@ -52,6 +52,8 @@ def email_text_body(mail)
             else
               part.decoded
             end
+          elsif part.content_type.include?('multipart/alternative')
+            part.parts.map(&:decoded)
```

Comment:
This would mean that `mail_bodies` can now become an array of arrays. I think we need to flatten it, so the utf8-encoding logic below works properly.